software_steering_council.md

```diff
@@ -0,0 +1,52 @@
+
+## Responsibilities
+
+### Sole responsibilities
+
+The SSC is responsible for the following:
+
+- Defining the submission, review, and approval process for Jupyter Enhancement Proposals.
+- Stewarding the JEP process to ensure that it is inclusive and participatory, and solicits feedback from the right stakeholders. As a starting point, [the NumFOCUS DISCOVER Cookbook](https://github.com/numfocus/DISCOVER-Cookbook) may offer some useful starting pointers.
+- Making decisions on JEPs after receiving community feedback (JEPs are the mechanism for elevating any software decision/idea up to the broader Project, while individual Subprojects can make decisions within their scope via their regular mechanisms).
+- Owning, managing and implementing the Jupyter Incubation process (https://github.com/jupyter-incubator).
+- Owning, managing and implementing the Jupyter Attic process (https://github.com/jupyter-attic).
+- Stewarding discussions and activities beyond JEPs that involve cross-cutting concerns, standards, protocols and other architectural issues that impact the entire project.
+- Owning and managing anything related to security vulnerabilities across the project, including the Jupyter security lists, any private security repositories, etc.
+
+### Shared responsibilities with the Board of Directors
+
+The SSC and BoD share the responsibility for:
+
+- Changing the Jupyter Governance model.
+- Creation of new Jupyter Subprojects.
+- Removal of Subprojects from Jupyter.
+- Designation of working groups that have a non-software focus but that carry a representative to the SSC, for specific topics where this is desired. This includes maintaining a working group on Inclusive Community Development; this group's representative can act as a point of contact on issues of Diversity, Inclusion and Community within the SSC and a bridge to the rest of the working group and the Board of Directors.
+
+## Membership
+
+SSC members are leaders from Jupyter Subprojects that wish to assist the community in sharing information across projects and participate in decision-making that affects many stakeholders in the Jupyter ecosystem. Membership requires active and ongoing participation in the project.
+
+### Member composition
+
+The Software Steering Council is composed of one representative from each of the Jupyter Subprojects. In addition, the SSC will contain members from certain working groups that have been created by the BoD and whose scope has important impact on SSC activities, such as Diversity and Inclusion, Accessibility, and Internationalization.
+
+### Term length
+
+There is no term length for SSC membership, as each Jupyter Subproject is in charge of selecting its own representative. Projects are encouraged to define their own expectations around SSC membership tenure, to share SSC membership responsibilities amongst core team members, and to provide mechanisms for healthy rotation that avoid burnout.
```

Comment:
Could we consider providing recommendations at least? I think of the ssc kind of like a board, and most healthy non profits have a good amount of turnover in their board to avoid groupthink and stale perspectives. I don’t think we want the exact same dynamics necessarily, but I think it would be helpful to create recommendations or processes that encourage dynamism in the SSC.

Reply:
It's a very good point, though there's challenges in what kind of ideas make sense across the board given the variability we have in subproject sizes and activity levels. Since we don't want to stall this PR for too long on implementation specifics, perhaps this is something we can address partly during the bootstrapping of these decision-making bodies across subprojects, and/or later in the life of the SSC itself. If you have more ideas on what we could put in now that's great, by all means send them in :) But for now I suggest we move forward and keep this in mind in the coming implementation weeks.

Side note: editing of that file will happen now on #103, since we just moved it over there.